src/common/SPELLS/shadowlands/conduits/rogue.js

```diff
@@ -1,6 +1,10 @@
 export default {
   //region Kyrian
-
+  BRONS_CALL_TO_ACTION: {
+    id: 333961,
+    name: `Bron's Call to Action`,
```

Comment:
'Bron\'s Call to Action'

Reply:
Would it be better to escape the extra ' ?

Reply:
github removed the \ from my comment

but yes, there should be a \ in front of the apostrophe